tegg/benchmark/http/run.sh

```
@@ -0,0 +1,34 @@
+#!/usr/bin/env bash
+
+echo
+npm start > output.log 2>&1 &
+pid=$!
```

Shellcheck (0.11.0)

[warning] pid appears unused. Verify use (or export if used externally).

(SC2034)